pkg/lint/lintersdb/builder_linter.go

```diff
@@ -286,9 +286,15 @@ func (b LinterBuilder) Build(cfg *config.Config) ([]*linter.Config, error) {
 			WithURL("https://github.com/golang/lint").
 			Deprecated("The repository of the linter has been archived by the owner.", "v1.41.0", "revive"),
 
+		linter.NewConfig(golinters.NewMND(&cfg.LintersSettings.Mnd)).
+			WithSince("v1.22.0").
+			WithPresets(linter.PresetStyle).
+			WithURL("https://github.com/tommy-muehle/go-mnd"),
+
 		linter.NewConfig(golinters.NewGoMND(&cfg.LintersSettings.Gomnd)).
 			WithSince("v1.22.0").
 			WithPresets(linter.PresetStyle).
+			Deprecated("The linter has been renamed.", "v1.58.0", "mnd").
```

Comment:
Why for `goerr13` you used alternative name without deprecation, but here you deprecated the old name? oO

Reply:
https://github.com/golangci/golangci-lint/pull/4562

Reply:
but why not `.WithAlternativeNames("gomd").` here?
I try to understand the reason of approaches difference

Reply:
https://github.com/golangci/golangci-lint/pull/4572/commits/623d457238670ee79767f1dcd63f5e9b82e0b519#diff-23eb637b7fd1abddf7f975546966746b84c387b8294f08c9a9a19cd7dee59082R12-R19